### DISCUSSION TITLE
How I can do Login as user from superadmin

### DISCUSSION THREAD

**Discussion by faigalizade** [filamentphp/filament#12861] [Help]
Hi everyone.
I have some roles (superadmin, admin, user) and UserResource.
I would like login as user from superadmin. How I can do it?
I have tried Filament Facade
`Filament::auth()->login($user)`
$user is instance of User(Authenticable and implements FilamentUser)

**Comment by leandrocfe** [comment]
Use https://github.com/404labfr/laravel-impersonate

        app(ImpersonateManager::class)->take(
            Filament::auth()->user(),
            $record,
            $guard
        );
